docs/style_guidelines.md

```diff
@@ -47,3 +47,6 @@ Under data, we can load the config file under its desired name, and we can attac
         name: {{ .Chart.Name }}-nginx
 
 This will put /etc/nginx/nginx.conf into the docker container with that absolute path, equivalent to copying over a local nginx.conf file in a Dockerfile. 
+
+## Git Hooks
+You may wish to include git hooks in your project, such as style checks or unit tests. It is recommended that you use this [Fivestars git-hooks](https://github.com/fivestars/git-hooks) tool to structure your custom hooks for better readability. We have set up a [.githooks](../.githooks) directory and written a mock [pre-commit](../.githooks/pre-commit-00-style) hook that simply prints a line and sleeps for 2 seconds. Follow the instructions for installing [git-hooks](https://github.com/fivestars/git-hooks) and try to make a commit to see it in action.
```

Comment:
hmm just thinking out loud - but since we will probably package the demo project with aladdin when open sourced, linking to a fivestars repo may or may not be something we want. Also if it is private, the user won't be able to see it. However, this may be a good thing since it would help our brand since the user will click on one of our repos.

Reply:
ya I was wondering too, should we open source git hooks too?

Reply:
The git hooks repo is already public :)